src/components/error-summary/error-summary.js

```diff
@@ -1,4 +1,6 @@
+import '../../vendor/polyfills/Function/prototype/bind'
 import '../../vendor/polyfills/Event' // addEventListener
+import '../../vendor/polyfills/Element/prototype/closest' // addEventListener
```

Comment:
Is this `// addEventListener` comment correct?

Reply:
Nope 😬